src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/OverseasEntitiesService.java

```diff
@@ -54,6 +57,28 @@ public OverseasEntitiesService(OverseasEntitySubmissionsRepository overseasEntit
         this.dateTimeNowSupplier = dateTimeNowSupplier;
     }
 
+    public boolean isSubmissionAnUpdate(String requestId, String overseasEntityId) throws SubmissionNotFoundException {
+        return getSubmissionType(requestId, overseasEntityId) == SubmissionType.UPDATE;
+    }
+
+    public SubmissionType getSubmissionType(String requestId, String overseasEntityId) throws SubmissionNotFoundException {
```

Comment:
I've kept this partly so that remove can be added (then not just yes/no). It's certainly a lot simpler without the FEATURE flag check which was not really in the right place (if needed at all).

Reply:
Makes sense to keep it. Wondering if method access modifier can/should be 'private' though, rather than 'public' - keep access as tight as possible. This then simplifies the unit-tests too (which seem to contain the only code that call this method, outside of this class).

Reply:
Sure, hoping package private is ok.